pkg/config.go

```diff
@@ -0,0 +1,36 @@
+package pkg
+
+
+// this object should be responsible for
+// loading file from external resources /xml/json/yaml
+type RulesContainer struct {
+	ruleMap map[string]map[string]int
+	timing map[string]int
+}
+
+func (rc *RulesContainer) getRule(rule string) map[string]int{
+	return rc.ruleMap[rule]
+}
+
+
+func InitRules() (RulesContainer) {
+	mm := map[string]map[string]int {
+		BASIC: {
+			ESPRESSO:3,
```

Comment:
Basic membership means 3 espressors. but per what? per 24 hours?
Is it understable from the code? and is it actually true?
I assume you were duplicating the example from README here, but then espresso maniac has `5 espressos per hour` (different time window)

Reply:
Yup here should be another map with configuration of time windows (per quota and product), but basically this map is containing only the amounts allowed per quota